app/javascript/stylesheets/sponsors-tile.css

```diff
@@ -15,16 +15,21 @@
     sm:flex-row
     justify-center
     items-center
+    max-w-sm
+    mx-auto
```

Comment:
This is the only actual style change -- I made the maximum div containing all the sponsor logos a bit smaller so it looks less sparse with only two :)